lib/info.js

```diff
@@ -54,6 +55,11 @@ exports.getBasicInfo = (id, options, callback) => {
       }
     }
 
+    const captchaForm = util.between(body, '<div class="g-recaptcha"', '>');
+    if(captchaForm) {
```

Comment:
if using a variable right away and only once, just put its value into where it's used

Reply:
Isn't it more readable to use it like this. 
In my opinion there shouldn't be to much text in an if.

Also it's easy to debug by adding a simple console.log between this lines. 

I read it in a style guide but have forgotten which one, i think it's very usefull.

Reply:
hmm that's a good point

Reply:
The problem with javascript is that since it's designed to be send over the web and interpreted live you (in general) want to reduce it as much as possible.
the performance should be about the same
removing the variable will end up with a line still easy to read and under 80 chars